%YAML:1.0
---
t2c_r:
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 9.98808205e-01, 1.90364514e-02, -4.49413545e-02,
          3.70598920e-02, -8.94991755e-01, 4.44540620e-01,
          -3.17596644e-02, -4.45676357e-01, -8.94630611e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 7.42197037e-01, -6.69912934e-01, 1.89792141e-02,
          -5.91047168e-01, -6.40946686e-01, 4.89745468e-01,
          -3.15922171e-01, -3.74705225e-01, -8.71658862e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -3.63603011e-02, -9.96504188e-01, 7.52151906e-02,
          -8.31135154e-01, 7.19452351e-02, 5.51396668e-01,
          -5.54880440e-01, -4.24650423e-02, -8.30845594e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -8.04201663e-01, -5.90744138e-01, -6.54297248e-02,
          -5.28717935e-01, 6.60752475e-01, 5.32788455e-01,
          -2.71508783e-01, 4.63063240e-01, -8.43715250e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -9.93105352e-01, 3.45526300e-02, 1.12017259e-01,
          9.26800370e-02, 8.16540599e-01, 5.69799781e-01,
          -7.17785656e-02, 5.76252997e-01, -8.14113200e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -9.93105352e-01, 3.45526300e-02, 1.12017259e-01,
          9.26800370e-02, 8.16540599e-01, 5.69799781e-01,
          -7.17785656e-02, 5.76252997e-01, -8.14113200e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -6.87278330e-01, 6.51814044e-01, -3.20604026e-01,
          2.39069402e-01, 6.19746625e-01, 7.47502506e-01, 6.85925841e-01,
          4.37095672e-01, -5.81767201e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -8.78669739e-01, 4.71402913e-01, 7.56228268e-02,
          4.55586731e-01, 7.80520916e-01, 4.28051233e-01, 1.42759398e-01,
          4.10568386e-01, -9.00584996e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -8.78669739e-01, 4.71402913e-01, 7.56228268e-02,
          4.55586731e-01, 7.80520916e-01, 4.28051233e-01, 1.42759398e-01,
          4.10568386e-01, -9.00584996e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -8.78669739e-01, 4.71402913e-01, 7.56228268e-02,
          4.55586731e-01, 7.80520916e-01, 4.28051233e-01, 1.42759398e-01,
          4.10568386e-01, -9.00584996e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -8.78669739e-01, 4.71402913e-01, 7.56228268e-02,
          4.55586731e-01, 7.80520916e-01, 4.28051233e-01, 1.42759398e-01,
          4.10568386e-01, -9.00584996e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -8.78669739e-01, 4.71402913e-01, 7.56228268e-02,
          4.55586731e-01, 7.80520916e-01, 4.28051233e-01, 1.42759398e-01,
          4.10568386e-01, -9.00584996e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -8.78669739e-01, 4.71402913e-01, 7.56228268e-02,
          4.55586731e-01, 7.80520916e-01, 4.28051233e-01, 1.42759398e-01,
          4.10568386e-01, -9.00584996e-01 ]
t2c_t:
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ -4.19700241e+01, 1.37121019e+01, 6.12000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ -1.44783798e+02, 6.53057861e+01, 6.08000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ -3.13288498e+01, 8.24893112e+01, 643. ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 1.23479034e+02, 2.06509132e+01, 600. ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.70564880e+01, -3.54079208e+01, 6.27000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.70564880e+01, -3.54079208e+01, 6.27000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.17596016e+01, -4.86222878e+01, 6.13000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.85949373e+00, 1.08124733e+02, 6.99000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.85949373e+00, 1.08124733e+02, 6.99000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.85949373e+00, 1.08124733e+02, 6.99000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.85949373e+00, 1.08124733e+02, 6.99000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.85949373e+00, 1.08124733e+02, 6.99000061e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.85949373e+00, 1.08124733e+02, 6.99000061e+02 ]
g2b_r:
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 9.96546268e-01, 8.20205957e-02, -1.29672335e-02,
          6.68842420e-02, -8.85372341e-01, -4.60046023e-01,
          -4.92140763e-02, 4.57589835e-01, -8.87800395e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 7.72231817e-01, -5.53049803e-01, -3.12720329e-01,
          -6.35213256e-01, -6.62208378e-01, -3.97472262e-01,
          1.27359210e-02, 5.05584776e-01, -8.62682939e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 8.95347726e-03, -8.17595780e-01, -5.75722992e-01,
          -9.98272538e-01, 2.61246655e-02, -5.26250079e-02,
          5.80665544e-02, 5.75199664e-01, -8.15949500e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -7.74871588e-01, -5.51437438e-01, -3.09015721e-01,
          -6.27444625e-01, 6.11633122e-01, 4.81890172e-01,
          -7.67280310e-02, 5.67293286e-01, -8.19933653e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -9.94275928e-01, 5.31308912e-02, -9.26953927e-02,
          -1.48731712e-02, 7.90312827e-01, 6.12523019e-01,
          1.05802245e-01, 6.10395610e-01, -7.84998775e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -8.16694081e-01, 4.67027187e-01, 3.38963628e-01,
          5.70710778e-01, 5.66704273e-01, 5.94251990e-01, 8.54396895e-02,
          6.78772330e-01, -7.29361534e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ -8.16695809e-01, 4.67027694e-01, 3.38958889e-01,
          5.70707858e-01, 5.66705227e-01, 5.94253957e-01, 8.54432881e-02,
          6.78771198e-01, -7.29362130e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 1.21407700e-03, 8.98623526e-01, 4.38718945e-01,
          9.98967230e-01, 1.88370831e-02, -4.13482673e-02,
          -4.54207137e-02, 4.38316047e-01, -8.97672534e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 8.16160142e-01, 4.47654426e-01, 3.65360290e-01,
          5.65429807e-01, -7.49001145e-01, -3.45378578e-01,
          1.19045019e-01, 4.88469839e-01, -8.64422083e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 9.96587574e-01, -7.53684714e-02, 3.36577557e-02,
          -4.74295318e-02, -8.56602192e-01, -5.13792872e-01,
          6.75550923e-02, 5.10443211e-01, -8.57253790e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 9.99657929e-01, 2.53623594e-02, 6.38346653e-03,
          2.60780267e-02, -9.85143185e-01, -1.69743344e-01,
          1.98353734e-03, 1.69851750e-01, -9.85467613e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 9.99657929e-01, 2.53620241e-02, 6.38540462e-03,
          2.60780249e-02, -9.85143304e-01, -1.69742718e-01,
          1.98551966e-03, 1.69851169e-01, -9.85467732e-01 ]
   - !!opencv-matrix
      rows: 3
      cols: 3
      dt: f
      data: [ 9.99657929e-01, 2.53620241e-02, 6.38540462e-03,
          2.60780249e-02, -9.85143304e-01, -1.69742718e-01,
          1.98551966e-03, 1.69851169e-01, -9.85467732e-01 ]
g2b_t:
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.06560730e+02, 5.18150696e+02, 5.86244690e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 8.10637024e+02, 4.13105682e+02, 5.60625793e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 8.46048096e+02, 2.91384430e+02, 5.55925293e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 7.09366882e+02, 1.53756927e+02, 5.60170227e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.03789307e+02, 7.87828827e+01, 5.77680420e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 3.96236816e+02, 5.99040985e+01, 5.60293335e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 3.96236237e+02, 5.99043350e+01, 5.60293274e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 4.81302246e+02, 2.55154144e+02, 5.23725708e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 4.73110229e+02, 5.19948120e+02, 5.23956177e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.25258911e+02, 5.71069397e+02, 5.35067871e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.17925598e+02, 2.15627853e+02, 5.60680908e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.17924805e+02, 2.15627594e+02, 5.60682251e+02 ]
   - !!opencv-matrix
      rows: 3
      cols: 1
      dt: f
      data: [ 6.17924805e+02, 2.15627594e+02, 5.60682251e+02 ]
